### ISSUE TITLE
Pkl weights conversion between stlyegan2-ada and stylegan?

### ISSUE THREAD

**Issue by 100330706** [NVlabs/stylegan2-ada#12]
Is it possible to convert the stylegan2 pkl weights to stylegan2-ada pkl weights and vicevers?

**Comment by woctezuma** [comment]
In my opinion, it depends on the config. Network architectures can differ if you don't use the right config.

**Comment by seawee1** [comment]
Shouldn't both networks in general be the same, as stylegan2-ada is more of a training technique rather than an architectural redesign of stylegan2?

**Comment by woctezuma** [comment]
You are correct. If you choose the right config, you will have the same network architecture as in StyleGAN2.
